package-managers/qubes-download-dom0-updates.sh

```diff
@@ -72,15 +72,15 @@ fi
 
 mkdir -p $DOM0_UPDATES_DIR/etc
 
-if [ -e /etc/debian_version ]; then
-    # Default rpm configuration on Debian uses ~/.rpmdb for rpm database (as
-    # rpm isn't native package manager there)
-    mkdir -p "$DOM0_UPDATES_DIR$HOME"
-    rm -rf "$DOM0_UPDATES_DIR$HOME/.rpmdb"
-    cp -r "$DOM0_UPDATES_DIR/var/lib/rpm" "$DOM0_UPDATES_DIR$HOME/.rpmdb"
+# Check if we need to copy rpmdb somewhere else
+DBPATH=`rpm -E '%{_dbpath}'`
+if [ ! "$DBPATH" = "/var/lib/rpm" ]; then
+    mkdir -p "$DOM0_UPDATES_DIR$DBPATH"
+    rm -rf "$DOM0_UPDATES_DIR$DBPATH"
```

Comment:
I suggest checking that `$DOM0_UPDATES_DIR` is something reasonable before running the `rm -rf`.

Reply:
note the rm -rf isnt a new part there, i just made the so-far debian-specific kludge more generic.
also, this is run as user, so the damage potential is ... limited.

Reply:
Good point!  Withdrawn.